src/build/env.py

```diff
@@ -141,27 +188,67 @@ def __enter__(self):  # type: () -> IsolatedEnvironment
         if 'PATH' in os.environ:
             exe_path.append(os.environ['PATH'])
 
-        self._replace_env('PATH', os.pathsep.join(exe_path))
+        '''
+        Place include, platinclude and LIBPL into our environment. They should
+        be present and may be used to compile native packages or similar.
+        '''
+        self._place_path_relative(sysconfig.get_path('include'))
+        self._place_path_relative(sysconfig.get_path('platinclude'))
+        self._place_path_relative(sysconfig.get_config_var('LIBPL'))
+
+        '''
+        We use PYTHONHOME to relocate the Python installation to our environment,
```

Comment:
You need the pyvenv.cfg only on python 3.4+. Copy on python 2 will not work. We should raise if that happens. Same on python 3 that's signed (macos+ windows store). We basically don't support those environments?

Reply:
Can you elaborate? We are not using `pyvenv.cfg`.